infrastructure/templates/public-api/bicepconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+  "experimentalFeaturesEnabled": {
+    "userDefinedTypes": true
+  }
+}
```

Comment:
Do we actually need this? I didn't think `userDefinedTypes` is an experimental feature any more

Reply:
IntelliJ Idea complained at me until I added this file and setting, so it's to satisfy the tooling that I added this.